src/lib/Contract.js

```diff
@@ -114,7 +113,10 @@ export default class Contract {
 
     return await Promise.all(logsArray.map(async log => {
       let parsedLog = await this.parseEvent(log);
-      parsedLog = this.formatLog(log, parsedLog);
+      if(parsedLog.name && parsedLog.eventFragment && parsedLog.eventFragment.inputs){
```

Comment:
`parsedLog.eventFragment && parsedLog.eventFragment.inputs` can be simplified to `parsedLog.eventFragment?.inputs`